content/en/docs/concepts/overview/working-with-objects/conditions.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Conditions
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+In Kubernetes, conditions as a concept are used to provide a detailed and structured status of a resource. This granular status information helps in understanding status of various Kubernetes objects and facilitates communication between different components of the system.
+
+<!-- body -->
+## Understanding Conditions
+
+A condition in Kubernetes includes several key fields:
+
+- **Type**: The specific kind of condition(e.g., `Ready`, `PodScheduled`).
+- **Status**: The state of the condition, which can be `True`, `False`, or `Unknown`.
+- **Reason**: String explaining why the condition is in a current state
+- **Message**: Message explaining details of the message
+- **Last Transition Time**: This represents the timestamp where condition last changed status
+
+## Example: Pod Conditions
+
+Pods, which are an integral part of the Kubernetes resources use conditions to indicate their state. Below is an example:
+
+```yaml
+status:
+  conditions:
+  - type: PodScheduled
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:24Z"
+  - type: Initialized
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:24Z"
+  - type: ContainersReady
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:31Z"
+  - type: Ready
+    status: "True"
+    lastTransitionTime: "2023-05-10T16:29:31Z"
+```
+
+In this example, each condition provides specific information about the pod's status, such as whether it has been scheduled, initialized, or is ready to handle traffic.
+
+In example above, each condition provides information as to the status of the pod. Whether the pod has been scheduled, initialized and ready to accept different traffic. Also whether the container in the pod are ready to be started.
+
+## Purpose and Usage of Conditions
```

Comment:
Please check the formatting of the list(s) in this section to ensure it renders correctly.
**[Preview link here for reference (click here)](https://deploy-preview-47149--kubernetes-io-main-staging.netlify.app/docs/concepts/overview/working-with-objects/conditions/#purpose-and-usage-of-conditions)**

Reply:
@dipesh-rawat  There seems to be some issue with the link you have mentioned. Can you reshare the correct ones